src/ape/managers/config.py

```diff
@@ -19,6 +33,65 @@ class DeploymentConfig(ConfigItem):
     contract_type: str
 
 
+class DependencyConfig(ConfigItem):
+    name: str
+    _compilers: "CompilerManager" = None  # type: ignore
```

Comment:
any dependencies that are not in the PM format require the compilers so they can put that way

(looking at you Github)

Not sure if there is a better way?